.github/workflows/netpol-cyclonus.yml

```diff
@@ -0,0 +1,83 @@
+name: Kind
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  KIND_VERSION: v0.9.0
+
+jobs:
+
+  check-changes:
+    name: Check whether tests need to be run based on diff
+    runs-on: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: vmware-tanzu/antrea/ci/gh-actions/has-changes@master
+        id: check_diff
+        with:
+          args: docs/* ci/jenkins/* *.md
+    outputs:
+      has_changes: ${{ steps.check_diff.outputs.has_changes }}
+
+  build-antrea-image:
```

Comment:
because there is a single job that actually consumes the image here, you don't need all that complexity and you don't need to upload the image as an artifact and worry about deleting it

You should only have 1 job here not 4:
- `check-changes` does not make sense for a CRON job
- `build-antrea-image` should become part of `test-netpol-cyclonus` (without the upload-artifact step)
- `artifact-cleanup` is not needed

Reply:
Okay that makes sense, thanks for pointing this out!

I mostly deduped this, quick question -- I don't need to `docker save` and `docker load` anymore right?

Reply:
that's correct, you can remove the commented-out lines. Glad the job configuration is much shorter now :)

Reply:
Same here :) 💯